src/components/Window.js

```diff
@@ -15,6 +15,8 @@ class Window extends Component {
    */
   render() {
     const { manifest, window } = this.props;
+    if (!window) return <></>;
```

Comment:
Maybe there's a better way to do this, but it seems like there's some asynchronous behavior that's making this hard to guard against. We clearly ran into this before in the mosaic layout by adding a guard clause like:

```
    const window = windows[id];
    if (!window) return null;
```

but, apparently, between that guard clause and the window container, state can change to remove the window.